eventdiscovery-sdk/src/main/res/values/theme_dark.xml

```diff
@@ -27,6 +27,7 @@
         <item name="schedjoules_cardBackgroundColor">@color/schedjoules_background_panel_dark</item>
         <item name="schedjoules_googleAttribution">@drawable/powered_by_google_dark</item>
         <item name="schedjoules_schedJoulesAttribution">@drawable/schedjoules_footer_powered_by_dark</item>
+        <item name="schedjoules_disabledButtonColor">@color/schedjoules_dark_disabled_button_background</item>
```

Comment:
The name is misleading. The button is not really disabled. It's just less prominent. Something like "reluctant" might be a better description. Alternatively we could try a regular non-colored button style (so the background of the butotn in the white theme is white as well). Maybe that's a better idea.